bot.rb

```diff
@@ -31,8 +31,8 @@
         when "/help"
           send.(HELLO)
         when "/story"
-          current_step = 0
-          check = true
+          CURENT_STEP = 0
+          CHECK = true
```

Comment:
- wrong usage of constance
- it's bad practice to set constants in the middle of method/class/whatever https://github.com/bbatsov/ruby-style-guide#consistent-classes